WordPress/src/main/res/values/strings.xml

```diff
@@ -1621,7 +1621,7 @@
     <string name="alert_error_adding_media">An error occurred while inserting media</string>
 
     <string name="stop_upload_dialog_title">Stop uploading?</string>
-    <string name="stop_upload_button">Stop Upload</string>
+    <string name="stop_upload_button" translatable="false">@string/ok</string>
```

Comment:
Can we simply use `android.R.string.ok` instead of having and extra `stop_upload_button` identifier?